src/app/Dashboard/Dashboard.tsx

```diff
@@ -68,6 +73,7 @@ export interface PropControl {
 export interface DashboardProps {}
 
 export interface DashboardCardProps {
+  dashboardIdx: number;
```

Comment:
Maybe call it `dashboardId`? In the future, we might want to avoid using index as id (i.e. sorting maybe).